### ISSUE TITLE
Launch console here not in context menu

### ISSUE THREAD

**Issue by NicTanghe** [microsoft/terminal#8641]
# Environment
```
Windows build number:  Win32NT 10.0.19042.0 Microsoft Windows NT 10.0.19042.0
Windows Terminal version (if applicable): preview from store
```

# Steps to reproduce

right click in explorer

# Expected behaviour
launch Terminal here is in the context menu.

# Actual behaviour
![image](https://user-images.githubusercontent.com/13624265/103016611-9848d480-4542-11eb-9856-2fc861b6111a.png)

It might be between keyboard and chair.

**Comment by KalleOlaviNiemitalo** [comment]
Duplicate of <https://github.com/microsoft/terminal/issues/6414>, blocked by a bug in Windows.

**Comment by zadjii-msft** [comment]
Yep, sure is. Thanks!

/dup #6414
